src/components/molecules/AddToDashboard.tsx

```diff
@@ -10,19 +10,19 @@ type AddToDashboardProps = {
 
 const AddToDashboard: React.FC<AddToDashboardProps> = (props) => {
     const dashboardItems = useReactiveVar(dashboardItemsVar);
-
     const added = useMemo(() => dashboardItems.some((el) => el.id === props.dashboardItemInfo.id), [dashboardItems]);
 
+    const updateStorage = (content: DashboardItemInfo[]) => {
+        dashboardItemsVar(content);
+        window.localStorage.setItem('dashboard', JSON.stringify(content));
+    };
+
```

Comment:
I would prefer if the localstorage modification was made somewhere else in the application with the use of a useEffect hook.

This way you could delete the visualisation from the dashboard and the localstorage entry would update "automatically", instead of either
a) it being updated the next time someone clicks this button
b) updated because someone added this code to another place (decreasing DRY-ness)

Reply:
Not sure if i'm understanding correctly. 
Should i create a function in utils or cache which updates the localstorage? 

Something like this?
```
useEffect(() => {
        updateLocalStorage(dashboardItems) //Some imported function
    }, [dashboardItems]);

    const addDashboardItem = () => {
        dashboardItemsVar([...dashboardItemsVar(), props.dashboardItemInfo]);
    };

    const removeDashboardItem = () => {
        dashboardItemsVar(dashboardItemsVar().filter((el) => el.id !== props.dashboardItemInfo.id));
    };

Reply:
Yes! Just don't put the useEffect in the molecule. I believe it would make sense to have this at the App.tsx (appx.).

Reply:
And when using code blocks, you can get syntax highlighting by typing the file extension after the three backticks;)